charts/global-network-policies/templates/allow-to-dns.yaml

```diff
@@ -15,6 +15,12 @@ spec:
       networking.gardener.cloud/to-dns: allowed
   egress:
   - to:
+    {{- if .Values.dnsServer }}
+    - ipBlock:
+        cidr: {{ .Values.dnsServer }}/32 # required for node local dns feature, allows egress traffic to kube-dns
+    {{- end }}
+    - ipBlock:
+        cidr: 169.254.20.10/32 # required for node local dns feature, allows egress traffic to node local dns cache
```

Comment:
Can you inject this as Helm value and use https://github.com/gardener/gardener/blob/6b183a89697dd64b88bdd4e60cc04a299d40bd16/pkg/operation/botanist/types.go#L33 to not duplicate the CIDR here?